Projects/Offer/Sources/Components/MainContentForm/Sections/Coverage/CoverageSection.swift

```diff
@@ -17,65 +17,40 @@ extension CoverageSection: Presentable {
 
 		let bag = DisposeBag()
 
+		let contentWrapper = UIStackView()
+		section.append(contentWrapper)
+
 		bag += state.quotesSignal.onValueDisposePrevious { quotes in
 			let innerBag = DisposeBag()
 
-			let innerSection = SectionView()
-			innerSection.dynamicStyle = .brandGrouped(
-				separatorType: .standard,
-				shouldRoundCorners: { _ in false }
-			)
-
 			if quotes.count > 1 {
-				let label = MultilineLabel(
-					value: "Read the full coverage of your insurances below.",
-					style: .brand(.body(color: .secondary))
-				)
-				let labelInset =
-					UIEdgeInsets(horizontalInset: 15, verticalInset: 0)
-					+ UIEdgeInsets(top: 0, left: 0, bottom: 15, right: 0)
-				innerBag += section.append(label.insetted(labelInset))
-
-				section.append(innerSection)
-
-				innerBag += {
-					innerSection.removeFromSuperview()
-				}
+				innerBag += contentWrapper.addArrangedSubview(MultiQuoteCoverage(quotes: quotes))
+			} else if quotes.count == 1 {
+				innerBag += contentWrapper.addArrangedSubview(SingleQuoteCoverage(quote: quotes[0]))
```

Comment:
Not that I think this can ever fail, but I have seen so many bugs come out of array[x] haha. It would probably be safer to write it as `else if let quote = quote.first { }` Feel free to leave it as is though.

Reply:
Totally agree, will fix 😄